scripts/package/aw-tauri.iss

```diff
@@ -0,0 +1,63 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "ActivityWatch"
+#define MyAppVersion GetEnv('AW_VERSION')
+#define MyAppPublisher "ActivityWatch Contributors"
+#define MyAppURL "https://activitywatch.net/"
+#define MyAppExeName "aw-tauri.exe"
+#define RootDir "..\.."
+#define DistDir "..\..\dist"
+
+#pragma verboselevel 9
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application. Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+; NOTE: the double {{ are used to escape the { character (needed for the AppId)
+AppId={{F226B8F4-3244-46E6-901D-0CE8035423E4}
```

Comment:
This needs to be updated to not collide with the aw-qt installation.

Should probably mention in MyAppName or AppVerName that this is about the Tauri version (not sure which is best).